.storybook/main.ts

```diff
@@ -80,7 +80,13 @@ const config: {
             }),
         );
 
-        return config;
+        return {
+            ...config,
+            cache: {
+                type: 'filesystem',
+                hashAlgorithm: 'sha256',
+            },
+        };
```

Comment:
```
config.cache = {
    ...config.cache,
    hashAlgorithm: 'sha256',
};

return config;
```
that way doesn't force the cache.type for both dev and prod modes. only sets the hash algorithm if one is used